co-authors-plus.php

```diff
@@ -385,7 +385,9 @@ public function coauthors_meta_box( $post ) {
 		?>
 
 		<div id="coauthors-edit" class="hide-if-no-js">
-			<p><?php _e( 'Click on an author to change them. Drag to change their order. Click on <strong>Remove</strong> to remove them.', 'co-authors-plus' ); ?></p>
+			<?php add_thickbox(); ?>
+			<a class="button thickbox" href="<?php echo get_bloginfo('url'); ?>/wp-admin/users.php?&page=view-guest-authorsTB_iframe=true&width=1200&height=640"><small><?php _e( 'Guest Authors', 'co-authors-plus' ); ?></small></a>
```

Comment:
Since the "Guest Authors" button isn't directly related to updating the styles, can we please remove it from this PR and open a new issue to discuss its addition? (I'm not a fan of using Thickbox here.)